### ISSUE TITLE
Week 1 of Atom post - what to include?

### ISSUE THREAD

**Issue by jbranchaud** [jbranchaud/splitting-atoms#42]
I am in the process of setting up a github pages/jekyll site with the content here so far (using the `gh-pages` branch of course). I wanted to add the first in a series of blog posts that will highlight important news about Atom, top packages for the week, best theme of the week, top tips and tricks, etc.

What three packages should I highlight in these week's post? Any tips and tricks to highlight (@jegtnes)? Anything else that belongs?

**Comment by rgbkrk** [comment]
I think it would be neat if you showed off packages that aren't currently "featured packages" on atom.io/packages, unless they truly warrant more publicity.

**Comment by jbranchaud** [comment]
@rgbkrk exactly! I want to show off cool stuff that might otherwise get lost in the noise.

**Comment by kenwheeler** [comment]
I think atom-script and mobile-preview are fine candidates! @jbranchaud @rgbkrk

**Comment by kenwheeler** [comment]
task-list too! Lol , bias aside, I really like the open recent package. Show in system is great too, and firepad.

**Comment by rgbkrk** [comment]
> I think atom-script and mobile-preview are fine candidates!

:+1: 

I certainly wasn't going to nominate the ones I've been involved in, but if you insist :wink:. Perhaps there's a way to gauge interest from the atom feed.

I'd certainly nominate @tombell's [test-status](https://github.com/tombell/test-status)

**Comment by jbranchaud** [comment]
Perhaps we could take this approach:
- Script Evaluation Packages: eval, runner, script
- Music Packages: atom-spotify, atom-rdio, ...
- One More Category: 1, 2, 3

In future weeks we could be a bit more selective, but seems like there is a lot worth mentioning this week.

**Comment by kenwheeler** [comment]
@rgbkrk re: test-status, wow that awesome, I hadn't even seen that one

**Comment by rgbkrk** [comment]
If we don't cover _everything_, it's ok. We'll have more content later.

**Comment by jegtnes** [comment]
Well, since you asked. ;) My favourites this week have been Bugsnag's shortcut cheat sheet https://bugsnag.com/blog/atom-editor-cheat-sheet, your guide on opening several projects in one window, the Block Travel package https://atom.io/packages/block-travel, and the guide on defining Atom commands in your init script: http://jasonrudolph.com/blog/2014/03/02/defining-atom-commands-in-your-init-script/

**Comment by karan** [comment]
- for highlighting packages.

Shameless plug. I published my first atom package a few days ago: https://atom.io/packages/atom-terminal
